### PR TITLE
[ADD][10.0]purchase_barcode:

### PR DESCRIPTION
**Pull request by kurniawanlucky**
* Add Product With Barcode at Purchase Order

### CODE REVIEW

File: purchase_barcode/README.rst [hunk 1]
```diff
@@ -0,0 +1,52 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License AGPL-3
+
+Purchase Order Barcode
+======================
+
+With Odoo <a href="https://github.com/kurniawanlucky/Odoo10.0">Purchase Barcode</a>,
```
Comment (andhit-r) [inline purchase_barcode/README.rst]: IMO, you should remove this or point it on OCA's link

File: purchase_barcode/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# © 2017
+#   @Lucky Kurniawan <kurniawanluckyy@gmail.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    "name": "Purchase Barcode",
+    "summary": "Add Product With Barcode at Purchase Order",
+    "version": "10.0.1.0.0",
+    "author": "Lucky Kurniawan ",
```
Comment (andhit-r) [inline purchase_barcode/__manifest__.py]: Add OCA as author

**Review by andhit-r — COMMENTED** [review]
@kurniawanlucky is there any way to test this module without barcode scanner?

File: purchase_barcode/README.rst (hunk 1 above)
Comment (kurniawanlucky) [reply purchase_barcode/README.rst]: yes, it`s done

File: purchase_barcode/__manifest__.py (hunk 2 above)
Comment (kurniawanlucky) [reply purchase_barcode/__manifest__.py]: yes, it`s done

**Comment by kurniawanlucky** [comment]
yes, i try this module without barcode scanner and running well.

**Pull request closed without merging**